src/PointwiseFunctions/GeneralRelativity/Christoffel.hpp

```diff
@@ -38,4 +45,84 @@ template <size_t SpatialDim, typename Frame, IndexType Index, typename DataType>
+                    tmpl::size_t<SpatialDim>, Frame>>;
+  static constexpr tnsr::ijj<DataType, SpatialDim, Frame> (*function)(
+      const tnsr::ijj<DataType, SpatialDim, Frame>&) =
+      &christoffel_first_kind<SpatialDim, Frame, IndexType::Spatial, DataType>;
+  using base = SpatialChristoffelFirstKind<SpatialDim, Frame, DataType>;
+  using type = typename base::type;
+};
+
+/// Compute item for spatial Christoffel symbols of the second kind
+/// \f$\Gamma^i_{jk}\f$ computed from the Christoffel symbols of the
+/// first kind and the inverse spatial metric.
+///
+/// Can be retrieved using `gr::Tags::SpacetimeChristoffelSecondKind`
+template <size_t SpatialDim, typename Frame, typename DataType>
+struct SpatialChristoffelSecondKindCompute
+    : SpatialChristoffelSecondKind<SpatialDim, Frame, DataType>,
+      db::ComputeTag {
+  using argument_tags =
+      tmpl::list<SpatialChristoffelFirstKind<SpatialDim, Frame, DataType>,
+                 InverseSpatialMetric<SpatialDim, Frame, DataType>>;
+  static constexpr Tensor<
+      DataType, Symmetry<2, 1, 1>,
+      index_list<SpatialIndex<SpatialDim, UpLo::Up, Frame>,
+                 SpatialIndex<SpatialDim, UpLo::Lo, Frame>,
+                 SpatialIndex<SpatialDim, UpLo::Lo, Frame>>> (*function)(
+      const Tensor<DataType, Symmetry<2, 1, 1>,
+                   index_list<SpatialIndex<SpatialDim, UpLo::Lo, Frame>,
+                              SpatialIndex<SpatialDim, UpLo::Lo, Frame>,
+                              SpatialIndex<SpatialDim, UpLo::Lo, Frame>>>&,
+      const Tensor<DataType, Symmetry<1, 1>,
+                   index_list<SpatialIndex<SpatialDim, UpLo::Up, Frame>,
+                              SpatialIndex<SpatialDim, UpLo::Up, Frame>>>&) =
+      &raise_or_lower_first_index<DataType,
+                                  SpatialIndex<SpatialDim, UpLo::Lo, Frame>,
+                                  SpatialIndex<SpatialDim, UpLo::Lo, Frame>>;
+  using base = SpatialChristoffelSecondKind<SpatialDim, Frame, DataType>;
+  using type = typename base::type;
+};
+
+/// Compute item for the trace of the spatial Christoffel symbols
+/// of the first kind
+/// \f$\Gamma_{i} = \Gamma_{ijk}g^{jk}\f$ computed from the
+/// Christoffel symbols of the first kind and the inverse spatial metric.
+///
+/// Can be retrieved using `gr::Tags::TraceSpatialChristoffelFirstKind`
+template <size_t SpatialDim, typename Frame, typename DataType>
+struct TraceSpatialChristoffelFirstKindCompute
+    : TraceSpatialChristoffelFirstKind<SpatialDim, Frame, DataType>,
+      db::ComputeTag {
+  using argument_tags =
+      tmpl::list<SpatialChristoffelFirstKind<SpatialDim, Frame, DataType>,
+                 InverseSpatialMetric<SpatialDim, Frame, DataType>>;
+  static constexpr Tensor<
+      DataType, Symmetry<1>,
+      index_list<SpatialIndex<SpatialDim, UpLo::Lo, Frame>>> (*function)(
+      const Tensor<DataType, Symmetry<2, 1, 1>,
```

Comment:
Any reason you can't use tensor aliases here?

Reply:
Nope. replaced with aliases.